resources/definitions/bfb.def.json

```diff
@@ -0,0 +1,40 @@
+{
+    "id": "bfb",
+    "name": "BFB",
+    "version": 2,
+    "inherits": "fdmprinter",
+    "metadata": {
+        "visible": true,
+        "author": "Ultimaker",
+        "manufacturer": "BFB",
+        "category": "Other",
+        "file_formats": "text/x-gcode",
+        "platform_offset": [ 0, 0, 0]
+    },
+
+    "overrides": {
+        "speed_topbottom": { "default_value": 40 },
+        "speed_print": { "default_value": 40 },
+        "machine_extruder_count": { "default_value": 1 },
+        "prime_tower_size": { "default_value": 7.745966692414834 },
```

Comment:
Where did you get these prime tower sizes? Did you test them? Legacy Cura had no such setting.